lib/theme.json

```diff
@@ -357,6 +357,78 @@
 					}
 				}
 			},
+			"core/form-input": {
+				"elements": {
+					"textInput": {
+						"border": {
+							"width": "1px",
+							"style": "solid",
+							"color": "#949494"
+						},
+						"typography": {
+							"lineHeight": "1.6",
+							"fontFamily": "inherit",
+							"fontSize": "1em"
+						},
+						"dimensions": {
+							"minHeight": "2em"
+						},
+						"spacing": {
+							"margin": {
+								"bottom": "0.5em"
+							},
+							"padding": {
+								"top": "0",
+								"right": "0.5em",
+								"bottom": "0",
+								"left": "0.5em"
+							}
+						}
+					}
+				}
+			},
+			"core/post-comments-form": {
```

Comment:
These are turning up as `:root :where(.wp-block-form-input textarea, .wp-block-form-input input:where([type="email"], .wp-block-form-input [type="number"], .wp-block-form-input [type="password"], .wp-block-form-input [type="search"], .wp-block-form-input [type="tel"], .wp-block-form-input [type="text"], .wp-block-form-input [type="tel"], .wp-block-form-input [type="url"]))` and are more specific than what the theme defines for some reason